<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="widgetResizable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>707</width>
           <height>536</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="imageLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionWork_Directory"/>
    <addaction name="actionLoad_calibration"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Zoom</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="action100"/>
    <addaction name="actionFit_to_window"/>
   </widget>
   <widget class="QMenu" name="menuColor">
    <property name="title">
     <string>Color</string>
    </property>
    <addaction name="actionRGB"/>
    <addaction name="actionGray"/>
    <addaction name="actionRed_channel"/>
    <addaction name="actionGreen_channel"/>
    <addaction name="actionBlue_channel"/>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuResolution">
    <property name="title">
     <string>Resolution</string>
    </property>
    <addaction name="action320_x_240"/>
    <addaction name="action640_x_480"/>
    <addaction name="action1024_x_768"/>
    <addaction name="action1280_x_1024"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSnapshoot"/>
    <addaction name="actionCalibrate"/>
    <addaction name="actionApply_undistort"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCamera"/>
   <addaction name="menuView"/>
   <addaction name="menuColor"/>
   <addaction name="menuResolution"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="action100">
   <property name="text">
    <string>100%</string>
   </property>
  </action>
  <action name="actionFit_to_window">
   <property name="text">
    <string>Fit to window</string>
   </property>
  </action>
  <action name="actionRGB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RGB</string>
   </property>
  </action>
  <action name="actionGray">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gray</string>
   </property>
  </action>
  <action name="actionRed_channel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Red channel</string>
   </property>
  </action>
  <action name="actionGreen_channel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Green channel</string>
   </property>
  </action>
  <action name="actionBlue_channel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Blue channel</string>
   </property>
  </action>
  <action name="actionAlpha_channel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Alpha channel</string>
   </property>
  </action>
  <action name="actionGradient">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gradient</string>
   </property>
  </action>
  <action name="action640_x_480">
   <property name="text">
    <string>640 x 480</string>
   </property>
  </action>
  <action name="action1024_x_768">
   <property name="text">
    <string>1024 x 768</string>
   </property>
  </action>
  <action name="action1280_x_1024">
   <property name="text">
    <string>1280 x 1024</string>
   </property>
  </action>
  <action name="action320_x_240">
   <property name="text">
    <string>320 x 240</string>
   </property>
  </action>
  <action name="actionSnapshoot">
   <property name="text">
    <string>Snapshoot</string>
   </property>
   <property name="shortcut">
    <string>Shift+S</string>
   </property>
  </action>
  <action name="actionWork_Directory">
   <property name="text">
    <string>Work Directory ...</string>
   </property>
  </action>
  <action name="actionCalibrate">
   <property name="text">
    <string>Calibrate ...</string>
   </property>
  </action>
  <action name="actionApply_undistort">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Apply undistort</string>
   </property>
  </action>
  <action name="actionLoad_calibration">
   <property name="text">
    <string>Load calibration ...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
